Dockerfile

```diff
@@ -1,9 +1,11 @@
 # Requires Docker v17.06 or later
-FROM golang:1.8.3 as builder
+FROM golang:1.8.5 as builder
```

Comment:
Any reason not to jump up to 1.9.2?

Reply:
I was just cautious. 😄 Shall I step it up to 1.9.2 then? Should I also increase to Alpine 3.6 at the same time?

Reply:
Ha, yea I think it is safe. 👍 for Alpine.